recipes/mimalloc/all/conanfile.py

```diff
@@ -144,6 +143,13 @@ def build(self):
             replace_in_file(self, os.path.join(self.source_folder, "CMakeLists.txt"),
                             "mimalloc-redirect.lib",
                             "mimalloc-redirect32.lib")
+        if self.version in ["1.7.9", "2.0.9"]:
+            replace_in_file(self, os.path.join(self.source_folder, "include", "mimalloc.h"),
```

Comment:
Can this be a patch please?

Reply:
@prince-chrismc 
I agree with you!
I create patches for it.